.circleci/config.yml

```diff
@@ -2,15 +2,18 @@ version: 2
 jobs:
   build:
     machine:
-      image: circleci/classic:latest
+      image: ubuntu-1604:201903-01
```

Comment:
Should we be concerned about moving from a `:latest` image to a particular version of an image?

Reply:
No.  [The circleci documentation](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images) makes clear that `ubuntu-1604:201903-01` is the recommended image to use.